test/parallel_api/numeric/numeric.ops/exclusive_scan_by_segment_perm_it.pass.cpp

```diff
@@ -139,6 +139,8 @@ test_exclusive_scan(sycl::queue q, std::vector<size_t>& perms, std::vector<TestV
     EXPECT_EQ_RANGES(expectedResults, results, "wrong effect from exclusive_scan_by_segment #2");
 }
 
+#define __ONEDPL_PERM_BASE_ITERATOR_HOST_DEVICE_POL_SUPPORT 0
```

Comment:
I am fine with that macro being in the test right now. Howerver, could you rename it? `__ONEDPL_PERM_BASE_ITERATOR_HOST_DEVICE_POL_SUPPORT` -> `_ONEDPL_PERM_BASE_ITERATOR_HOST_DEVICE_POL_SUPPORT` so there is only one leading underscore as in other internal macros. 

But in the long term, it might be useful to keep it inside the primary headers as a feature macro. e.g. as #945 suggests.